pmd-core/src/main/java/net/sourceforge/pmd/AbstractPropertySource.java

```diff
@@ -127,25 +118,22 @@ public boolean hasDescriptor(PropertyDescriptor<?> descriptor) {
         return result;
     }
 
-    /**
-     * {@inheritDoc}
-     */
+
     @Override
     public <T> void setProperty(PropertyDescriptor<T> propertyDescriptor, T value) {
```

Comment:
The use of `Arrays.asList` everywhere to set the values is very boilerplate-y.  I wonder if this additional signature would be useful:
```
public <T> void setProperty(PropertyDescriptor<List<T>> propertyDescriptor, T... values)
```
Or, maybe, assuming the more specific interface is used in practice:
```
public <T> void setProperty(MultiValuePropertyDescriptor<T> propertyDescriptor, T... values)
```